<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>新建列</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>新建列</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>新建列</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="ZTabWidget" name="tabWidget">
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_W"/>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_U"/>
    <addaction name="action_P"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_U_2"/>
    <addaction name="separator"/>
    <addaction name="action_T"/>
    <addaction name="action_C"/>
    <addaction name="action_P_2"/>
    <addaction name="action_L"/>
    <addaction name="separator"/>
    <addaction name="action_Bing"/>
    <addaction name="action_F"/>
    <addaction name="action_N_2"/>
    <addaction name="action_V"/>
    <addaction name="action_R"/>
    <addaction name="action_G"/>
    <addaction name="separator"/>
    <addaction name="action_A_2"/>
    <addaction name="action_D"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="action_W_2"/>
    <addaction name="action_F_2"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menu_Z">
     <property name="title">
      <string>缩放(&amp;Z)</string>
     </property>
     <addaction name="action_I"/>
     <addaction name="action_O_2"/>
     <addaction name="action_R_Z"/>
    </widget>
    <addaction name="menu_Z"/>
    <addaction name="action_S_2"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_H"/>
    <addaction name="action_F_3"/>
    <addaction name="separator"/>
    <addaction name="action_A_3"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_N">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_W">
   <property name="text">
    <string>新窗口(&amp;W)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>打开(&amp;O)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>另存为(&amp;A)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_U">
   <property name="text">
    <string>页面设置(&amp;U)...</string>
   </property>
  </action>
  <action name="action_P">
   <property name="text">
    <string>打印(&amp;P)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="action_U_2">
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="action_T">
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_P_2">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="action_L">
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
  </action>
  <action name="action_Bing">
   <property name="text">
    <string>使用Bing搜索...</string>
   </property>
  </action>
  <action name="action_F">
   <property name="text">
    <string>查找(&amp;F)...</string>
   </property>
  </action>
  <action name="action_N_2">
   <property name="text">
    <string>查找下一个(&amp;N)</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>查找上一个(&amp;V)</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>替换(&amp;R)...</string>
   </property>
  </action>
  <action name="action_G">
   <property name="text">
    <string>转到(&amp;G)...</string>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>时间/日期(&amp;D)</string>
   </property>
  </action>
  <action name="action_W_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="action_F_2">
   <property name="text">
    <string>字体(&amp;F)...</string>
   </property>
  </action>
  <action name="action_S_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="action_H">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="action_F_3">
   <property name="text">
    <string>发送反馈(&amp;F)</string>
   </property>
  </action>
  <action name="action_A_3">
   <property name="text">
    <string>关于ZNote(&amp;A)</string>
   </property>
  </action>
  <action name="action_Menu">
   <property name="text">
    <string>&amp;Menu</string>
   </property>
  </action>
  <action name="action_I">
   <property name="text">
    <string>放大(&amp;I)</string>
   </property>
  </action>
  <action name="action_O_2">
   <property name="text">
    <string>缩小(&amp;O)</string>
   </property>
  </action>
  <action name="action_R_Z">
   <property name="text">
    <string>恢复默认缩放</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZTabWidget</class>
   <extends>QTabWidget</extends>
   <header>../widgets/tab_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
